package.json

```diff
@@ -78,5 +78,8 @@
         "env": "production"
       }
     ]
-  }
+  },
+  "browserslist": [
```

Comment:
This stops the regeneratorRuntime error that @Link2Twenty was getting. By adding this modern browserlist, regeneratorRuntime is not generated in the compiled JS code since async/await does not need to be polyfilled.

@Link2Twenty, I've tested this locally, but always good to have another set of eyes on this. Remove the unpacked extension and load it unpacked again. The error should not occur when the extension loads and runs the snippet of async/await code.

Reply:
I was still having the same issue with this config but I've tried changing it locally to this

```js
"browserslist": [
  "defaults",
  "not IE 11"
]
```

and that seemed to work.